djqscsv/djqscsv.py

```diff
@@ -4,24 +4,32 @@
 
 from django.core.exceptions import ValidationError
 from django.utils.text import slugify
-from django.http import HttpResponse
+from django.http import HttpResponse, StreamingHttpResponse
```

Comment:
From what I can tell, `StreamingHttpResponse` is in all of the supported Django versions (even 1.6). We could probably drop support for 1.6 and 1.7 too, given that 1.7 is unsupported as of the end of Q4 2015.

See: https://www.djangoproject.com/download/#supported-versions